src/System.Text.Json/tests/Serialization/Default.WriteTests.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class DefaultTests
```

Comment:
Based on API review, we might also need more tests for default value handling, e.g.
- on deserialization
- for collection properties on classes/structs
- for collection elements
- for nested collections